iidm/iidm-modification/src/main/java/com/powsybl/iidm/modification/topology/CreateCouplingDeviceBuilder.java

```diff
@@ -16,9 +16,13 @@ public class CreateCouplingDeviceBuilder {
     private String busOrBbsId2 = null;
 
     private String switchPrefixId = null;
+    private NamingStrategy namingStrategy = null;
 
     public CreateCouplingDevice build() {
-        return new CreateCouplingDevice(busOrBbsId1, busOrBbsId2, switchPrefixId);
+        if (namingStrategy == null) {
+            namingStrategy = new DefaultNamingStrategy();
+        }
```

Comment:
You could have initialized `namingStrategy` with its default value at its declaration (just like as in the other builder).